__tests__/user/user.query.test.js

```diff
@@ -582,3 +586,166 @@ describe('Testing obtaining information restrictions', () => {
     expect(data.isSuccess).toEqual(true);
   });
 });
+
+describe('Filter users', () => {
+  let adminToken;
+  let usersId;
+
+  beforeAll(async () => {
+    await testUsersSet.map(user => createUser(user).catch(e => e));
+    adminToken = await loginAdmin(superAdminUser);
+
+    const users = await getAllUsers(adminToken).then(res =>
```

Comment:
Don`t use **then** structure - just use the function in it for **users** variable